modules/client-configuration/pages/autoinst-pxeboot.adoc

```diff
@@ -0,0 +1,99 @@
+. the distribution is installed automatically according to a {kickstart} or an {ay};
+. the profile calls a code snippet that registers the client to the {productname} Server, either as a traditional client, or as a Salt minion.
+
+image::cobbler_menu.png[scaledwidth=80%]
+
+The installation server can either be the {productname} Server or one of its proxies. To install from a proxy, you must synchronize the TFTP tree between the server and the proxy.
+
+The DHCP server may also provide other configuration information to the client, like a host name, the address of a router and the address of a domain name server. Some of this information might be needed for the autoinstallation, for example if you specify the installation server by its domain name.
+
+In the installation menu, the first of the choices is [guimenu]``Local boot``. Selecting this choice continues the boot process from the local disk drives. This option is selected automatically if no distribution was selected after a certain time. This is a security measure to prevent from starting an autoinstallation if no human operator is there to select one of the distributions.
+
+Alternatively, the installation might start automatically from one of the distributions without manual intervention. This is called "unattended provisioning".
+
+The "bare metal" functionality is a variant of unattended provisioning where the bootloader file only registers the client at the {productname} Server and does not start the installation. You may then trigger a reinstallation as described in previous section xref:client-configuration:autoinst-reinstall.adoc[Reinstalling Registered Systems].
+
+.Procedure: Performing an Installation with PXE Boot
+. Prepare the DHCP server, see xref:client-configuration:autoinst-pxeboot.adoc#prepare-the-dhcp-server[Prepare the DHCP Server].
+. Prepare an autoinstallation distribution, see xref:client-configuration:autoinst-distributions.adoc[Autoinstallation Distributions].
+. Prepare an autoinstallation profile, see xref:client-configuration:autoinst-profiles.adoc[Autoinstallation Profiles].
+. Reboot the client, and chose a distribution to install if needed.
+
+Some other steps are optional:
+
+* if you want to use a proxy as the installation server, see xref:client-configuration:autoinst-pxeboot.adoc#synchronize-the-tftp-tree-with-proxies[Synchronize the TFTP Tree with Proxies];
+* if you desire a fully unattended provisioning, see xref:client-configuration:autoinst-xxxxx.adoc[].
+
+
+[[prepare-the-dhcp-server]]
+== Prepare the DHCP Server
+
+The PXE boot process uses DHCP to find the TFTP server. The {productname} Server or its proxies can act as such a TFTP server.
+
+You must have administrative access to the network’s DHCP server. Edit the DHCP configuration file so that it points to the {productname} Server as the TFTP boot server:
```

Comment:
```suggestion
The PXE boot process uses DHCP to find the TFTP server. 
The {productname} Server or its proxies can act as such a TFTP server.
You must have administrative access to the network DHCP server. 
To do this, you can edit the DHCP configuration file so that it points to the {productname} Server as the TFTP boot server.
```

Reply:
ok